usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -58,6 +43,13 @@ SELECT
 
   vw_es_award_search.awarding_agency_id,
   vw_es_award_search.funding_agency_id,
+  (
+    SELECT a1.id
+    FROM agency a1
+    WHERE a1.toptier_agency_id = (SELECT a2.toptier_agency_id FROM agency a2 WHERE a2.id = vw_es_award_search.funding_agency_id)
+    ORDER BY a1.toptier_flag DESC, a1.id
+    LIMIT 1
+  ) AS funding_toptier_agency_id,
```

Comment:
> Would it make sense to build one or two VIEWs in the database that capture this repetitive sql?

Perhaps it would. This comment is a little outdated now, not sure it is in-scope with this PR anymore

> 1. ...

I don't recall why it is different off-the-top-of-my-head. I do remember working on the SQL w/ Kirk

> 2. ...does the order by capture that by putting "TRUE" items first?

Yes